weatherApp/App/NavigationService.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  NavigationService.swift
+//  weatherApp
+//
+//  Created by Dario Nikolic on 17/08/2020.
+//  Copyright © 2020 Dario Nikolic. All rights reserved.
+//
+
+import UIKit
+
+class NavigationService {
+    
+    private let appDependencies = AppDependencies()
+    private let navigationController = UINavigationController()
+    private var window: UIWindow!
+    
+    init(window: UIWindow) {
+        presentInWindow(window: window)
+        pushHomeViewController()
+    }
+    
+    func pushHomeViewController() {
```

Comment:
We usually name this as `goToHome` or `presentHomeScreen` or something like that - whoever calls this is not interested on how we do it (do we use `push` or `set` or that we use `ViewControllers`). It is a job for `NavigationService` to know those details.